soft4pes/control/mpc/controllers/__init__.py

```diff
@@ -1,3 +1,8 @@
+"""
+Model predictive control for power electronic systems.
```

Comment:
Let's have:

Model predictive control (MPC) algorithms and solvers for power electronic systems